.github/workflows/release.yml

```diff
@@ -24,7 +24,7 @@ jobs:
 
     - uses: actions/setup-node@v2
       with:
-        node-version: '14.x'
+        node-version: 18.x
```

Comment:
what is the point here?

Reply:
what's the difference from which `node.js` to publish in npm?